### PR TITLE
Update codelab with Fountain generators

### PR DESCRIPTION
**Pull request by zckrs**
Related to "[Ship a fresh new codelab for 2016 🍹](https://github.com/yeoman/yeoman.github.io/issues/599)" I finally submit PR who rewrites [codelad](http://yeoman.io/codelab/) with the main entry of Fountain generators ([generator-fountain-webapp](https://github.com/FountainJS/generator-fountain-webapp)).

***

* Let's Scaffold a Web App with Yeoman: _remove occurrences of Grunt and Bower, replace sentence's AngularJS by React and Angular_
* ~~Meet Yeoman~~: _delete from codelab_
* Step 1: Set up your dev environment: _prerequisites Nodejs v4, remove bower grunt-cli install_
* Step 2: Install a Yeoman generator: _replace generator-angular by generator-fountain-webapp_
* Step 3: Use a generator to scaffold out your app: _proposed without imposing stack React + Webpack + Babel + SASS_
* Step 4: Review the Yeoman-generated app: _quick tour of main folders and files_
* Step 5: Preview your app in the browser: _replace `grunt serve`command by `npm run serve`_
* ~~Step 6: Start writing our AngularJS application~~: _delete from codelab_
* ~~Step 7: Use Bower to install packages~~: _delete from codelab_
* Step 6: Test with Karma and Jasmine: _remove AngularJS code, describe how Karma is congifured, present `npm test` and `npm run test:auto`_
* Step 7: Make Todos persistent with local storage
* Step 8: Get ready for production
* Keep going: _link to Yeoman, Fountain, React, Angular2, Webpack, JSPM_

### CODE REVIEW

**Comment by SBoudrias** [comment]
Wow thanks a lot for sencindg this!! 😄

**Comment by sindresorhus** [comment]
@zckrs Awesome!

Do you know any other Fountain maintainers or users that could review this?

**Comment by zckrs** [comment]
I think @Swiip and @micaelmbagira could review. Maybe @delapouite too.

File: app/codelab/scaffold-app.md [hunk 1]
```diff
@@ -48,42 +44,24 @@ If you have a few generators installed, you'll be able to interactively choose f
...
-
-![](/assets/img/codelab/image_8.png)
-
-Next, you are prompted to select what Angular modules you would like to include as well:
-
-![](/assets/img/codelab/image_9.png)
-
-Angular modules are self-contained JavaScript files with helpful functionality. For example, the ngResource module (*angular-resource.js*) provides interaction support with RESTful services.
+The FountainJS generator provides some choices to use your favorite:
+* framework ([React](https://facebook.github.io/react/), [Angular2](https://angular.io/) or [Angular1](https://angularjs.org/))
```
Comment (micaelmbagira) [inline app/codelab/scaffold-app.md]: The list is not rendering as a list
capture d ecran 2016-06-13 a 09 53 38

Comment (zckrs) [inline app/codelab/scaffold-app.md]: Fixed https://github.com/yeoman/yeoman.github.io/pull/653/commits/8cf3630ed7d8989b233996c671042baeecd1f66c

File: app/codelab/preview-inbrowser.md [hunk 2]
```diff
@@ -8,38 +8,38 @@ To preview your web app in your favourite web browser, you don't have to do anyt
...
 To this instantly:
 
-![](/assets/img/codelab/image_14.png)
+![](/assets/img/codelab/05_after_live_reload.png)
 
 <p class="codelab-paging">
   <a href="index.html#toc">&laquo; Return to overview</a>
   or
-  <a href="write-app.html">Go to the next step &raquo;</a>
+  <a href="run-unit-test.html">Go to the next step &raquo;</a>
```
Comment (micaelmbagira) [inline app/codelab/preview-inbrowser.md]: Should be `run-unit-tests.html`

Comment (zckrs) [inline app/codelab/preview-inbrowser.md]: Fixed https://github.com/yeoman/yeoman.github.io/pull/653/commits/38e067860d578384b1c36472bae030d901338d10

**Comment by micaelmbagira** [comment]
LGTM 👍

**Comment by Swiip** [comment]
To be honest, we already reviewed internally before going here, so of course LGTM 👍 ;)

**Comment by SBoudrias** [comment]
Alright, so based on my reading, this looks good!

I don't really have time to go through and try everything, but I'm confident for a merge. - I know you guys will be around if issues are brought up by the community. Let's :shipit:

**Comment by SBoudrias** [comment]
Also, I'll prepare a tweet for this :)

**Comment by zckrs** [comment]
Thank you Simon 😊.

**Comment by SBoudrias** [comment]
Just found out there's a broken image here http://yeoman.io/codelab/local-storage.html - the rest looked fine after deploy!

**Comment by zckrs** [comment]
https://github.com/yeoman/yeoman.github.io/pull/656 

Fixed. :)

**Comment by addyosmani** [comment]
Thank you so so soooo much for this. We've needed a fresh codelab for a while. Really appreciate the great work here.

**Comment by addyosmani** [comment]
One suggestion: at the top of http://yeoman.io/codelab/index.html we mention the old credits for the original codelab. Worth updating to reflect the new author :)

**Comment by zckrs** [comment]
@addyosmani 

>Thank you so so soooo much for this. We've needed a fresh codelab for a while. Really appreciate the great work here.

Always a pleasure helping Yeoman :smiley: really.

>Worth updating to reflect the new author.

:grin: This could be the consecration of my life. :grin: 

Do you want I edit that in my fix-PR https://github.com/yeoman/yeoman.github.io/pull/656 or or do you prefer to do it yourself ?

**Comment by addyosmani** [comment]
@zckrs Feel free to in a new PR :) or we can do it. It's a quick one minute fix either way :)

**Pull request merged**